test/test-clusters/src/main/java/org/elasticsearch/test/cluster/FeatureFlag.java

```diff
@@ -15,7 +15,8 @@
  * to indicate that this feature is required and should be enabled when appropriate.
  */
 public enum FeatureFlag {
-    TIME_SERIES_MODE("es.index_mode_feature_flag_registered=true", Version.fromString("8.0.0"), null);
+    TIME_SERIES_MODE("es.index_mode_feature_flag_registered=true", Version.fromString("8.0.0"), null),
+    INFERENCE_RESCORER("es.inference_rescorer_feature_flag_enabled", Version.fromString("8.10.0"), null);
```

Comment:
Is this missing an `=true`? This PR seems to have caused #100362, so I think there's _something_ different to before about the way the feature flag is set in release builds.

Reply:
I think you're right. I'll get this fixed.

Reply:
https://github.com/elastic/elasticsearch/pull/100366